index.html

```diff
@@ -1,199 +1,172 @@
-                            <li class="down" ><section class="ic"><section class="jsfile"></a><a class="css3">CSS</a></section></section></li></li>
-                            <li class="down"><section class="ic"><a class="lang">Frameworks</a><img class=" vec" src="vect1.png"></section></li>
-                            <li class="down"><section class="ic"><img class="line" src="line.png"></section></li>
-                            <li class="down"><section class="ic"><a class="lang">Skills</a><img class=" vec" src="vect1.png"></section></li>
-                            <li class="down"><section class="ic"><img class="line" src="line.png"></section></li>
-                        </ul>
-                    </section>
-                </section>
-                </section>
-                </section>
-
-            </main>
-        <footer></footer>
-    </body>
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" type="text/css" href="styles.css" /> 
+        <title>
+            Portfolio project!!!
+        </title>
+    </head>
+    <body>
+                <section class="page">
+                <section class="card">
+                    <section class="part-2">
+                    <section class="part-2">
+                        <img class="Portfolio" src="Portfolio.png" alt="Feyisa">
+                    </section>
+                    <section class="part-2">
+                        <h1 class="tonic">Tonic</h1>
+                    </section>
+                    <section class="part-2" >
+                    <ul class="can">
+                        <li class="can"><a class="canopy">CANOPY</a></li>
+                        <li class="can"><img class="counter" src="point.png"></li>
+                        <li class="can" ><a class="back" >Back End Dev</a></li>
+                        <li class="can"><img class="counter" src="point.png"></li>
+                        <li class="can" ><a class="num">215</a></li>
+                    </ul>
+                    </section>
+                    <section class="part-2"><p class="daily">A daily selection of privately 
+                        personalized reads; no accounts or 
+                        sign-ups required.</p>
+                    </section>
+                    <section class="part-2">
+                        <section class="but">
+                            <section class="buthtml"><a class="htmlbut">HTML</a></section>
+                            <section class="butcss"><a class="cssbut" type="button">CSS</a></section>
+                            <section class="butjs"><a class="jsbut" type="button">JavaScript</a></section>
+                        </section></section>
+                    <button class="see">See project</button>

```

Comment:
- [ ] i kindly suggest that you implement the hover and onClick styling to the button tag as the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=XO3IYDMPH9pYDTCx-0)